sdks/python/apache_beam/metrics/monitoring_infos.py

```diff
@@ -0,0 +1,216 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# cython: language_level=3
+# cython: profile=True
+
+from __future__ import absolute_import
+from apache_beam.portability.api import beam_fn_api_pb2
+from apache_beam.metrics.cells import DistributionData
+from apache_beam.metrics.cells import DistributionResult
+from apache_beam.metrics.cells import GaugeResult
+from apache_beam.metrics.cells import GaugeData
+from google.protobuf import timestamp_pb2
+
+import time
+
+USER_COUNTER_URN_PREFIX = 'beam:metric:user:'
+ELEMENT_COUNT_URN = 'beam:metric:element_count:v1'
+START_BUNDLE_MSECS_URN = (
+    'beam:metric:pardo_execution_time:start_bundle_msecs:v1')
+PROCESS_BUNDLE_MSECS_URN = (
+    'beam:metric:pardo_execution_time:process_bundle_msecs:v1')
+FINISH_BUNDLE_MSECS_URN = (
+    'beam:metric:pardo_execution_time:finish_bundle_msecs:v1')
+TOTAL_MSECS_URN = (
+    'beam:metric:ptransform_execution_time:total_msecs:v1')
+
+# TODO(ajamato): Implement the remaining types, i.e. Double types
+# Extrema types, etc. See:
+# https://s.apache.org/beam-fn-api-metrics
+SUM_INT64_TYPE = 'beam:metrics:SumInt64'
```

Comment:
Can we stick with the existing case of snake_case instead of CamelCase for all URNs?

Also, Eugene was able to move the URNs into proto form (https://github.com/apache/beam/pull/4672) to share across languages. Can we adopt the same now?

Reply:
Done